app/server/server.js

```diff
@@ -81,14 +83,15 @@ server.get('*', (req, res, next)=> {
 
       getReduxPromise().then(()=> {
         let reduxState = escape(JSON.stringify(store.getState()))
+        let metaHeader = Helmet.rewind();
```

Comment:
I think we should `rewind` in the `catch` block as well to avoid [possible memory leak](https://github.com/nfl/react-helmet#server-usage).